deploy/restartComponent.sh

```diff
@@ -8,7 +8,14 @@
 
 HOST=$(hostname)
 DATENOW=$(date +%s)
-DEST_NAME=cms-wmcore-team
+# look-up alert emails from WMA secret file
+EMAIL_ALERT_WM=`grep EMAIL_ALERT_WM $WMA_SECRETS_FILE`
```

Comment:
I am inclined to say that this should be a single variable with email address space separated.

In addition, you also need to `sed` or replace the content of this grep, otherwise it won't contain only the email address(ess).

Reply:
ok, used single email address variable